runtime/src/main/java/org/corfudb/runtime/collections/DiskBackedCorfuTable.java

```diff
@@ -152,11 +159,19 @@ public DiskBackedCorfuTable(@NonNull PersistenceOptions persistenceOptions,
         try {
             this.statistics = new Statistics();
             this.statistics.setStatsLevel(StatsLevel.ALL);
-            rocksDbOptions.setStatistics(statistics);
-            persistenceOptions.getWriteBufferSize().map(rocksDbOptions::setWriteBufferSize);
+            this.rocksDbOptions.setStatistics(statistics);
+            persistenceOptions.getWriteBufferSize().map(this.rocksDbOptions::setWriteBufferSize);
+
+            final BlockBasedTableConfig tableConfig = new BlockBasedTableConfig();
+            if (persistenceOptions.isDisableBlockCache()) {
+                tableConfig.setNoBlockCache(true);
+            } else {
+                persistenceOptions.getBlockCache().map(tableConfig::setBlockCache);
```

Comment:
nit: perhaps a comment explaining that if no blockCache related options have been set then this would default to the 64MB block cache with the default tableConfig?